tahoe_scorm/storages.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from exeptions import ScormException
+
+def tahoe_scorm_storage(xblock):
+    """
+    Multi-tenant SCORM storage backend for overhangio/openedx-scorm-xblock .
+    """
+    from django.conf import settings
+    from django.core.files.storage import get_storage_class
+    from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+    from openedx.core.djangoapps.appsembler.api.sites import get_site_for_course
+
+    current_site = get_site_for_course(xblock.course_id)
+    current_organization = current_site.organizations.first()
+    sub_folder = configuration_helpers.get_value_for_org(
+        current_organization.name,
+        "SCORM_DIR",
+        False
+    )
```

Comment:
This is very slow because it does string search instead of foreinkey lookup, please change it to:

```suggestion
    site_config = SiteConfiguration.objects.get(site=site)
    sub_folder = site_config.get_value('SCORM_DIR', False)
```

Reply:
Done.